modules/mobile/spec/requests/mobile/v1/health/immunizations_spec.rb

```diff
@@ -334,83 +326,38 @@
 
     describe 'when multiple items have same date' do
       context 'date is available' do
-        it 'returns items in alphabetical order by group name' do
+        it 'sorts items in date order then alphabetical order by group name' do
```

Comment:
This test and the next were not well described and it was difficult to understand what they were testing, so I made some changes.